pages/your_domain/human_data.md

```diff
@@ -1,6 +1,6 @@
 ---
 title: Human data
-description: Data management solutions for human data.
+description: Key data management considerations solutions for projects with human data.
```

Comment:
considerations solutions, I think one of the two needs to go :)

Reply:
@pinarpink What do you think? :)

Reply:
@bedroesb  yeah, we can drop the "solutions."

Reply:
Great!